scipy/sparse/linalg/_matfuncs.py

```diff
@@ -33,19 +33,19 @@ def inv(A):
 
     Parameters
     ----------
-    A : (M,M) ndarray or sparse matrix
+    A : (M, M) sparse matrix
         square matrix to be inverted
 
     Returns
     -------
-    Ainv : (M,M) ndarray or sparse matrix
+    Ainv : (M, M) sparse matrix
         inverse of `A`
 
     Notes
     -----
     This computes the sparse inverse of `A`. If the inverse of `A` is expected
     to be non-sparse, it will likely be faster to convert `A` to dense and use
-    scipy.linalg.inv.
+    `numpy.linalg.inv`.
```

Comment:
why change this? In general I'd say we prefer `scipy.linalg`, because it's a superset of `numpy.linalg` and it may have more optimized routines in some cases. Plus also because these are the scipy docs:)

Reply:
adding the backticks is useful though, to make this a cross-link

Reply:
Ah, my bad. I saw `sparse.linalg.inv`, so I changed "scipy.linalg.inv" into “`numpy.linalg.inv`”. Exactly, it should be `scipy.linalg.inv`.

Reply:
I restored the `scipy.linalg.inv` and added the backticks. Thanks for your review.